File: servidor/src/main/java/org/example/serverautoescuela/domain/Incidencia.java
package org.example.serverautoescuela.domain;

import lombok.*;

import java.time.LocalDateTime;

@Builder
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Incidencia {
    private int id;
    private String titulo;
    private String descripcion;
    private String ubicacion;
    private LocalDateTime fecha;
    private String dniProfesor;
    private String matricula;
    private boolean resuelta;
}

File: appmovil/app/src/main/java/com/example/autoescuelapp/domain/Incidencia.kt
package com.example.autoescuelapp.domain

import java.time.LocalDateTime

data class Incidencia(
    val id: Int = 0,
    val tituloIncidencia: String,
    val descripcion: String,
    val ubicacion: String,
    val fecha: LocalDateTime = LocalDateTime.now(),
    val dniProfesor: String,
    val matriculaCoche: String,
)
